### ISSUE TITLE
NVDA doesn't always start properly with windows

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/test_nvdaTracImport#141]
**Reported by cnorman on 9 Apr 2008 07:49 UTC**
Hi All,
I set NVDA to start with windows by putting a shortcut to it in the %userprofile%\start menu\programs\startup folder, and sometimes it doesn't work when I log in. Other times it just makes the startup sound, but nothing else, and yet other times (fewer than more), it starts fine.

Just thought I'd better log this, I have done a search for it in the rest of trac, and can find nothing.

HTH.

Chris Norman.

**Comment by michaelDCurran** [comment]
**Comment by aleksey_s on 9 Apr 2008 07:59 UTC**
Chris, when you setting "version" field to 0.5 do you mean this problem occurs in nvda 0.5? as you know, 0.5 not supported anymore. if not so, please assign "trunk" value to version field.

**Comment by michaelDCurran** [comment]
**Comment by mdcurran on 27 Apr 2008 10:39 UTC**
Replying to [cnorman](ticket:62):
Chris, can you provide any more info for this bug?
For instance:
*If NVDA fails to start, can you still start it manually?
*Is it perhaps only starting on a reboot, or perhaps a cold startup?
*Do you hear the NVDA error sound after the NVDA start sound when it does get that far?
*What other items do your have in your startup folder, or what items do you have set to start when Windows starts? perhaps there is one that is failing before NVDA and possibly not allowing NVDA to start.

Could you also set your NVDA log level to debug, and then the next time NVDA fails to start (but does play the start sound at least) can you save the log file and attach it to this ticket? perhaps some more info can be gotten from there, if your log level is debug of course.
The log file will be in your user temp dir.

**Comment by michaelDCurran** [comment]
**Comment by jteh on 2 May 2008 04:36 UTC**
Reporter has not commented as to whether this is a problem in trunk and this works fine for Mick in trunk. 0.5 is no longer being maintained, so closing as wontfix. If this does not work in trunk, please reopen and set version to trunk.
